pkg/api/postage.go

```diff
@@ -487,6 +487,12 @@ func (s *Service) postageTopUpHandler(w http.ResponseWriter, r *http.Request) {
 			jsonhttp.PaymentRequired(w, "out of funds")
 			return
 		}
+		if errors.Is(err, ErrNotImplemented) {
+			s.logger.Debug("topup batch: not implemented", "error", err)
+			s.logger.Error(nil, "topup batch: not implemented")
+			jsonhttp.NotImplemented(w, "not implemented")
```

Comment:
You can pass the error here directly and the `jsonhttp` will take care of it. The same is below.

Reply:
If you do `jsonhttp.NotImplemented(w, nil)`, this will automatically translate using the `http.StatusText(http.StatusNotImplemented)` to "Not Implemented".